### ISSUE TITLE
client can't connect to OSS InfluxDB

### ISSUE THREAD

**Issue by davidgs** [influxdata/influxdb-client-js#166]
## Expected Behavior

When using the client library, I should be able to connect to InfluxDB 2 OSS, but this is not possible.

## Current Behavior

[Error] Credentials flag is true, but Access-Control-Allow-Credentials is not "true".
[Error] Fetch API cannot load <my influxDB OSS instance> due to access control checks.
[Error] Failed to load resource: Credentials flag is true, but Access-Control-Allow-Credentials is not "true". (query, line 0)

## Steps/Code to Reproduce the Problem

1. step1
1. ...

```
// ...
writeApi.writeRecord(...)
```

## Specifications

- Version:
pulled client from Master. 
InfluxDB OSS is latest beta build.
- Platform:

**Comment by sranka** [comment]
Thank you for reporting this issue, #160 fixes the CORS problem on the js client side. Still, there is another CORS issue on the server side (influxd) that must be fixed in order to make this work.

**Comment by bednar** [comment]
The new OSS version of InfluxDB with https://github.com/influxdata/influxdb/pull/17640 is released. Please try it again with our latest nightly build: 

```bash
npm i @influxdata/influxdb-client@1.2.0-nightly.10
```

**Comment by sranka** [comment]
@bednar it also requires to test against a nightly build of influxdb, the fix on the server side will be since v2.0.0-beta.9; works for me
